drafts/2016-12-13-this-week-in-rust.md

```diff
@@ -47,39 +47,25 @@ If you are a Rust project owner and are looking for contributors, please submit
-* [HIR: Separate signatures from function bodies](https://github.com/rust-lang/rust/pull/37918)
-* [Obligations are now evaluated in LIFO order](https://github.com/rust-lang/rust/pull/38059)
-* [Support `?Sized` in `where` clauses](https://github.com/rust-lang/rust/pull/37791)
-* [New `type_size_limit` crate attribute](https://github.com/rust-lang/rust/pull/37789)
-* [`target_feature` attribute](https://github.com/rust-lang/rust/pull/38079)
-* [`unmarked_api` feature removed](https://github.com/rust-lang/rust/pull/38087)
-* [`copy_from_slice(_)` got faster for small slices](https://github.com/rust-lang/rust/pull/37573)
-* [`String::split_off(..)`](https://github.com/rust-lang/rust/pull/38056)
-* [show short multiline spans in full](https://github.com/rust-lang/rust/pull/37369)
-* [SUpport macro invocation paths (e.g. `foo::bar!(..)`)](https://github.com/rust-lang/rust/pull/38082)
-* [Cargo will now correctly retry downloading in case of network error](https://github.com/rust-lang/cargo/pull/3348)
-* [Cargo has release branches now](https://github.com/rust-lang/cargo/pull/3345)
-* [Cargo: Fixed SSL paths](https://github.com/rust-lang/cargo/pull/3342) (this broke downloading for some days in nightly)
+74 pull requests were [merged in the last week][merged]. This contains a good number of plugin-breaking changes.
+
+[merged]: https://github.com/issues?q=is%3Apr+org%3Arust-lang+is%3Amerged+merged%3A2016-12-05..2016-12-12
+
+* [Use link(kind) annotation to fix native Windows imports](https://github.com/rust-lang/rust/pull/37973) ([RFC #1717](https://github.com/rust-lang/rfcs/blob/master/text/1717-dllimport.md))
+* [Function arity errors now show the original definition](https://github.com/rust-lang/rust/pull/38121)
+* [`HashMap`/-`Set` now allocate smarter on `extend_from_iter(_)`](https://github.com/rust-lang/rust/pull/38017)
+* [Faster `sort()`](https://github.com/rust-lang/rust/pull/38192)
+* [More forwarded `ExactSizeIterator` / `is_empty()`](https://github.com/rust-lang/rust/pull/38149)
+* [Stricter lifetimes for `LateLintPass`](https://github.com/rust-lang/rust/pull/38191) (Plugin-unbreaking 🙂)
+* [`Iterator::nth(_)` no longer needs `self` to be `Sized`](https://github.com/rust-lang/rust/pull/38134)
+* [Incremental compilation will now recompile items on visibility change](https://github.com/rust-lang/rust/pull/38272)
+* [`-Z always_encode_mir`](https://github.com/rust-lang/rust/pull/38217)
+* [dylib symbol handling improvements](https://github.com/rust-lang/rust/pull/38117)
+* [`tidy` now checks the licenses of vendored dependencies](https://github.com/rust-lang/rust/pull/38291)
+* [Rustbuild is now the default build system](https://github.com/rust-lang/rust/pull/37817) (1 year in the making)
+* [Allow `--test` on procedural-macro crates](https://github.com/rust-lang/rust/pull/38107)
+* [Cargo now picks up `build.rs` scripts by default](https://github.com/rust-lang/cargo/pull/3361) (unless `package.build = false` explicitly)
```

Comment:
Cool but they didn't pull the trigger on it. Cargo will pick them up in the future, for now an orphan build.rs is a warning (and no build script).